pre-receive-hooks/reject-commits.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+#
+# Reject certain commits from being pushed to the repository
+#
+# This can be a useful pre-receive hook [1] if you rewrote the history
+# of a repository and you want to ensure nobody pushes the old commits
+# again.
+#
+# Usage: Add the commits you want to reject in the
+#        "<list commit hashes here>" below.
+#
+# [1] https://help.github.com/en/enterprise/2.16/user/articles/working-with-pre-receive-hooks
```

Comment:
I suggest we remove the version number from the documentation. Doing so allows the URL to automatically redirect to the latest version of the documentation, so we won't need to update the link in the future.

```suggestion
# [1] https://help.github.com/en/enterprise/user/articles/working-with-pre-receive-hooks
```

The downside to this is that if the implementation changes for some reason (low risk in this case), this script's URL would point to the new implementation.

Let me know which way you want to go with it, I'm 👍 with either.

Reply:
Nice!! I didn't know that this is possible. Thank you 👍